integration_test/ops_agent_test.go

```diff
@@ -2672,6 +2672,27 @@ func TestLoggingAgentCrashRestart(t *testing.T) {
 	})
 }
 
+func TestLoggingFluentbitSelfLogs(t *testing.T) {
+	t.Parallel()
+	gce.RunForEachPlatform(t, func(t *testing.T, platform string) {
+		t.Parallel()
+		ctx, logger, vm := agents.CommonSetup(t, platform)
+
+		if err := setupOpsAgent(ctx, logger, vm, ""); err != nil {
+			t.Fatal(err)
+		}
+
+		// Wait long enough for the data to percolate through the backends
+		// under normal circumstances. Based on some experiments, 2 minutes
+		// is normal; wait a bit longer to be on the safe side.
+		time.Sleep(3 * time.Minute)
```

Comment:
Is this actually necessary in practice? `gce.WaitForLog` already waits for 400 seconds...

Reply:
Good catch! Will remove